tmt/steps/__init__.py

```diff
@@ -370,23 +454,27 @@ def delegate(cls, step, data):
         """
         # Filter matching methods, pick the one with the lowest order
         for method in cls.methods():
+            assert 'how' in data and data['how'] is not None
```

Comment:
I'm not sure this `assert` is needed, `data` is `Dict[Any, Any]`, mypy shouldn't make any waves when `data['how']` is requested...

Unless it complains about `startswith(data['how'])` where the argument must be a string. I'm not sure this happens *after* the validation or *before*, if before, we cannot use `assert` here.

@psss would you know in what stage is this code called? Is it after JSON schema is used? That should happen when tmt is creating test/plan/story classes, validation should be called transparently making fmf data loaded into these objects are valid. Does that cover use of `data` in this place?

Reply:
Hmm, I tried to remove that, and looks like it was not needed. Maybe it was resolved somewhere after I added the line.

Reply:
assert is removed. It was not needed